tests/compile/test_fusion.py

```diff
@@ -24,16 +24,25 @@ def __init__(self, hidden_size: int, eps: float, *args, **kwargs):
             torch.rand(hidden_size, hidden_size).to(dtype=FP8_DTYPE).t()
             for _ in range(2)
         ]
+        self.cutlass_fp8_supported = cutlass_fp8_supported()
 
     def forward(self, x):
         resid = torch.relu(x)
         y = self.norm[0](x)
 
-        x2 = apply_fp8_linear(y, self.w[0], self.scale[0], self.scale[1])
```

Comment:
Can you make `cutlass_fp8_supported` a parameter (`True` and `False` on CUDA, `False` otherwise)? That way we test both paths on CUDA as well. And we can explicitly skip the CUDA non-cutlass path for now until I fix the case with padding

Reply:
Actually, nevermind - I can add that case once I add the fix.